lib/util/SortableSet.js

```diff
@@ -1,5 +1,7 @@
 "use strict";
-
+//TODO: Make this a generic type
```

Comment:
https://github.com/webpack/webpack/pull/7046

Reply:
Should I just wait on this to land. Thats my thought.

Reply:
I'll work on it tomorrow